pkg/fqdn/dnsproxy/proxy.go

```diff
@@ -834,7 +835,8 @@ func (p *DNSProxy) CheckAllowed(endpointID uint64, destPortProto restore.PortPro
 
 	for selector, regex := range epAllow {
 		// The port was matched in getPortRulesForID, above.
-		if regex != nil && selector.Selects(destID) && (regex.String() == matchpattern.MatchAllAnchoredPattern || regex.MatchString(name)) {
+		// versioned.AllHandle() always gets the latest version of selectors
```

Comment:
nit: `AllHandle` -> `Latest` and then maybe the comment is no longer necessary

Reply:
Removed, thanks!